gtwrap/pybind_wrapper.py

```diff
@@ -320,17 +353,23 @@ def wrap_namespace(self, namespace):
             module_var = self._gen_module_var(namespaces)
 
             if len(namespaces) > len(self.top_module_namespaces):
-                wrapped += (' ' * 4 + 'pybind11::module {module_var} = '
-                            '{parent_module_var}.def_submodule("{namespace}", "'
-                            '{namespace} submodule");\n'.format(
-                                module_var=module_var,
-                                namespace=namespace.name,
-                                parent_module_var=self._gen_module_var(namespaces[:-1]),
-                            ))
+                wrapped += (
+                    ' ' * 4 + 'pybind11::module {module_var} = '
+                    '{parent_module_var}.def_submodule("{namespace}", "'
+                    '{namespace} submodule");\n'.format(
+                        module_var=module_var,
+                        namespace=namespace.name,
+                        parent_module_var=self._gen_module_var(
+                            namespaces[:-1]),
+                    ))
 
+            # Wrap an include statement, namespace, class or enum
             for element in namespace.content:
                 if isinstance(element, parser.Include):
-                    includes += ("{}\n".format(element).replace('<', '"').replace('>', '"'))
+
+                    includes += ("{}\n".format(element).replace('<',
+                                                                '"').replace(
+                                                                    '>', '"'))
```

Comment:
autoformat and unnecessary extra line

Reply:
Again formatter result.

Reply:
See above comment - I thought this was unintentional; feel free to disregard